examples/Hello Wall/hello-wall-add-specific-fire-rating.ifcx

```diff
@@ -0,0 +1,33 @@
+{
+  "header": {
+    "id": "ifc5.technical.buildingsmart.org/examples/Hello Wall/hello-wall-add-fire-rating-60@v1.ifcx",
+    "version": "ifcx_alpha",
+    "author": "authorname",
+    "timestamp": "2025-06-27"
+  },
+  "imports": [],
+  "schemas": {
+    "bsi::ifc::prop::FireRating": {
+      "value": {
+          "dataType": "Enum",
+          "enumRestrictions": {
+            "options": ["R30", "R60"]
+          }
+      }
+    }
+  },
+  "data": [
+    {
+      "path":"25503984-6605-43a1-8597-eae657ff5bea",
+      "attributes": {
+        "bsi::ifc::prop::FireRating": "R30"
+      }
+    },
+    {
+      "path":"ab143723-f7b1-5368-b106-55896e88d768/My_Project/My_Site/My_Building/My_Storey/Wall/Window_001",
```

Comment:
what's the difference between this path and simply: "592504dc-469a-44d6-9ae8-c801b591679b" (the Window_001 instance)?